src/app/modules/form/admin-form/__tests__/admin-form.issue.controller.spec.ts

```diff
@@ -0,0 +1,323 @@
+import { ObjectId } from 'bson-ext'
+import { StatusCodes } from 'http-status-codes'
+import { errAsync, okAsync } from 'neverthrow'
+import { PassThrough } from 'stream'
+
+import dbHandler from '../../../../../../__tests__/unit/backend/helpers/jest-db'
+import expressHandler from '../../../../../../__tests__/unit/backend/helpers/jest-express'
+import { FormIssueMetaDto } from '../../../../../../shared/types'
+import { IPopulatedForm, IPopulatedUser } from '../../../../../types'
+import * as AuthService from '../../../auth/auth.service'
+import { DatabaseError } from '../../../core/core.errors'
+import * as IssueService from '../../../issue/issue.service'
+import { MissingUserError } from '../../../user/user.errors'
+import * as UserService from '../../../user/user.service'
+import { ForbiddenFormError, FormNotFoundError } from '../../form.errors'
+import * as AdminFormIssueController from '../admin-form.issue.controller'
+import { PermissionLevel } from '../admin-form.types'
+
+jest.mock('../../../user/user.service')
+const MockUserService = jest.mocked(UserService)
+jest.mock('src/app/modules/auth/auth.service')
+const MockAuthService = jest.mocked(AuthService)
+jest.mock('src/app/modules/issue/issue.service')
+const MockIssueService = jest.mocked(IssueService)
+
+describe('admin-form.issue.controller', () => {
+  beforeAll(async () => await dbHandler.connect())
+  beforeEach(async () => {
+    await dbHandler.clearDatabase()
+  })
+  afterAll(async () => await dbHandler.closeDatabase())
+
+  describe('handleGetFormIssue', () => {
+    beforeEach(() => {
+      jest.clearAllMocks()
+    })
+    const MOCK_FORM_ID = new ObjectId().toHexString()
+    const MOCK_USER_ID = new ObjectId()
+    const MOCK_USER = {
+      _id: MOCK_USER_ID,
+      email: 'notarealuser@example.com',
+    } as IPopulatedUser
+    const MOCK_FORM = {
+      admin: MOCK_USER as IPopulatedUser,
+      _id: MOCK_FORM_ID,
+      title: 'form title',
+    } as IPopulatedForm
+    const MOCK_REQ = expressHandler.mockRequest({
+      params: { formId: MOCK_FORM_ID },
+      session: {
+        user: {
+          _id: MOCK_USER_ID,
+        },
+      },
+    })
+
+    it('should return issues as request is processed successfully', async () => {
```

Comment:
/nit should return issues `as -> when` request is ...

We usually use `when` which describes the arrange section

Reply:
done https://github.com/opengovsg/FormSG/pull/6516/commits/dad6758c7ee8e83eb6e270976ea80879b5a3c8c5